apps/website/docs/react/api/shorthands.md

```diff
@@ -0,0 +1,176 @@
+---
+sidebar_position: 3
+---
+
+# shorthands
```

Comment:
would be nice to have a link to `limitations.md` file

Reply:
Added 👍